private_dot_config/nvim/lua/lsp/clients/prettier.lua

```diff
@@ -0,0 +1,11 @@
+local prettier = require("prettier")
+
+prettier.setup({
+  bin = 'prettier',
+  filetypes = {
+    "javascript",
+    "javascriptreact",
+    "typescript",
+    "typescriptreact"
```

Comment:
and ruby?

Reply:
mmmm... yep

Reply:
I'm going to handle `rbprettier` in another PR